examples/connectivity/plot_adhd_tangent.py

```diff
@@ -0,0 +1,107 @@
+"""
+Comparing different connectivity measures
+=========================================
+
+This example compares different measures of functional connectivity between
+regions of interest.
+
+"""
+
+# Fetch dataset
+import nilearn.datasets
+atlas = nilearn.datasets.fetch_atlas_msdl()
+dataset = nilearn.datasets.fetch_adhd()
+
+# Extract and preprocess regions time series
+import nilearn.input_data
+import joblib
+mem = joblib.Memory('/home/sb238920/CODE/Parietal/nilearn/nilearn_cache/adhd')
+masker = nilearn.input_data.NiftiMapsMasker(
+    atlas.maps, resampling_target="maps", detrend=True,
+    low_pass=None, high_pass=None, t_r=2.5, standardize=False,
+    memory=mem, memory_level=1)
```

Comment:
My bad. I thought that when you specify the cache, memory_level=1 was
forced. Leave it then but it looks like an issue to me (nilearn issue, not
yours of course).

On Thu, Jul 16, 2015 at 12:43 PM, salma1601 <notifications@github.com>
wrote:

> In examples/connectivity/plot_adhd_tangent.py
> <https://github.com/nilearn/nilearn/pull/677#discussion_r34775249>:
>
> > +
> > +"""
> > +
> > +# Fetch dataset
> > +import nilearn.datasets
> > +atlas = nilearn.datasets.fetch_atlas_msdl()
> > +dataset = nilearn.datasets.fetch_adhd()
> > +
> > +# Extract and preprocess regions time series
> > +import nilearn.input_data
> > +import joblib
> > +mem = joblib.Memory('/home/sb238920/CODE/Parietal/nilearn/nilearn_cache/adhd')
> > +masker = nilearn.input_data.NiftiMapsMasker(
> > +    atlas.maps, resampling_target="maps", detrend=True,
> > +    low_pass=None, high_pass=None, t_r=2.5, standardize=False,
> > +    memory=mem, memory_level=1)
>
> why not specify memory_level ? I have an ugly warning when not
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nilearn/nilearn/pull/677/files#r34775249>.
>